Fixes #1 -- Added Contribution Guidelines.

prahe184 opened the pull request:
**Description**
#Added CONTRIBUTING.md with all the important points stated in the issue.
#Incorporated references from various open-source guidelines.
#Followed best practices from GitHub documentation to ensure clarity and completeness.

**Changes Made**
#Change <> into ``Project Lead``.
#Added a Reporting Unexpected Behavior Section.


Closes #1

dr-rompecabezas commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+   - Example: If you are working on a bug fix for login, you might name your branch `bugfix-login-issue`.
+
+### 4. **Make Your Changes**  
+   - Ensure your code adheres to the project's style guide and best practices.
```
Our style has yet to be defined or evolve, but it's still good to have a reference to style here.

dr-rompecabezas commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+
+### 6. **Document Your Changes**  
+   - If you are adding a new feature, update the documentation to reflect your changes.
+   - Add relevant usage examples or instructions in the `README.md` or any relevant docs.
```
We might want to add Sphinx to the project. We could open an issue and see if anyone is willing to work on this.

dr-rompecabezas replied:
I'll open an issue with some details.

dr-rompecabezas replied:
Here's the issue, in case you're interested in exploring this further:
https://github.com/OmdenaAI/GibdetColombiaChapter_UrbanTreeObservatory/issues/4

prahe184 replied:
> I'll open an issue with some details.

Ok Sir

dr-rompecabezas commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+   - In your PR description, clearly describe what you've done, why it's necessary, and include any relevant issue number (e.g., `Fixes #123`).
+
+   **Image Example**:  
+   ![Create PR](![alt text](image-2.png))
```
Same note as before re: images.

prahe184 replied:
Yes Sir, I will make changes Sir

dr-rompecabezas commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+
+## Code Contribution Standards
+
+### **Protecting the `main` Branch**
```
The main branch protection can be enforced with a ruleset in GitHub, but it's still great to have it here.

prahe184 replied:
Shall I add a link of Github docs Like for more info you can check out -- link of github docs Sir?

dr-rompecabezas replied:
No, leave this as-is @prahe184.

@juanpac96 if you agree, please create the rule to protect the main branch.

prahe184 replied:
> No, leave this as-is @prahe184.
> 
> @juanpac96 if you agree, please create the rule to protect the main branch.

I have added GitHub docs Link Sir

dr-rompecabezas commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+-Submit Your First PR: When you're ready, submit a PR and follow the review process.
+### **Code of Conduct**
+This project and everyone participating in it is governed by the OmdenaAI/GibdetColombiaChapter_UrbanTreeObservatory Code of Conduct.
+By participating, you are expected to uphold this code. Please report unacceptable behavior to <>.
```
We may not want to display an email publicly here. How about...

Please report unacceptable behavior to the project lead?

And we leave it up to contributors to use Slack, DM, email.

prahe184 replied:
Yes Sir, I've gone through the changes required and Now, I will be working on them Sir

dr-rompecabezas reviewed (changes requested):
Looks great!

There is only one comment that I believe should be addressed before merging. It's at the end of the file, to replace the `<>` placeholder with `project lead` or similar.

prahe184 commented on .github/CONTRIBUTING.md:
```diff
@@ -0,0 +1,120 @@
+     If the repository URL is `https://github.com/OmdenaAI/GibdetColombiaChapter_UrbanTreeObservatory`, after forking it, the URL will become `https://github.com/YOUR-USERNAME/GibdetColombiaChapter_UrbanTreeObservatory`.
+
+   **Image Example**:  
+      ![Fork Button](![alt text](image-1.png))
```
I will make sure this time to create a separate folder for images and then it makes referencing them easier.

dr-rompecabezas commented on .github/images/Forkimage.png.png:
Hmmm... I don't think it's a good idea to store images in the `.github` directory.

I'm also thinking these images can be a distraction from the core content as they take up significant space.

What do you think about removing the sample images altogether?

prahe184 replied:
Ok I will remove and update my PR Sir

dr-rompecabezas commented on .github/images/image.png:
Same as above. What do you think about removing the images altogether?

prahe184 replied:
Removed Sir

prahe184 commented:
Hey, @dr-rompecabezas Is the issue resolved?

dr-rompecabezas reviewed (changes requested):
To summarize my review, we should probably not store images under `.github` and now that I see the images I worry they might be too large and distracting to include alongside the text. Thank you.

prahe184 commented:
> Thank you for removing the references to the images from the markdown document. Please delete the two images in a new commit and push again.

I have already removed images folder and removed images from .github/ directory Sir
